packages/babel-plugin-transform-typescript/src/index.ts

```diff
@@ -229,9 +229,23 @@ export default declare((api, opts) => {
                 continue;
```

Comment:
Can you rebase on latest `main`? We should also register global type for type-only import specifiers so they can be removed in the following cases:

```ts
import { type A } from "module";
export { A }
```